spec/service/integration_test_path_finder_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'appmap/service/integration_test_path_finder'
+
+describe AppMap::Service::IntegrationTestPathFinder do
+  subject { described_class }
+
+  describe '.count' do
+    it 'counts existing paths' do
```

Comment:
These tests could pass even if the code is broken, couldn't they?

Might be better to use something like https://github.com/fakefs/fakefs to verify that `service` is really working as expected.

Reply:
@apotterri I adjusted the service so it can be tested with unit tests. Please have a look.